0.download-data/README.md

```diff
@@ -0,0 +1,31 @@
+# Download CFReT data
+
+The information regarding the CFReT pilot data are as follows:
+
+```
```

Comment:
I bet that we can start standardizing these metadata :)

Can you think of some standard indicators? Please review this highlight in Nature Methods: https://www.nature.com/collections/djiciihhjh

👆 this will help with deciding which parameters to use, and how to best format. Our lab should be expert in this, so it is worth careful documentation and working towards complete understanding.